docs/reference/search/search-your-data/behavioral-analytics/behavioral-analytics-overview.asciidoc

```diff
@@ -0,0 +1,78 @@
+[[behavioral-analytics-overview]]
+== Search Analytics
```

Comment:
Because this is no longer a product name, I would not capitalize "Analytics".
```suggestion
== Search analytics
```

Reply:
💯